managed-databases/redis/api-cli/managing-username-and-password.mdx

```diff
@@ -0,0 +1,125 @@
+---
+meta:
+  title: Managing the default user's username and password
+  description: Learn more about the default user's username and password for Managed Database for Redis™
+content:
+  h1: Managing the default user's username and password
+  paragraph: Learn more about the default user's username and password for Managed Database for Redis™
+tags: databases user redis username password
+categories:
+  - managed-databases
+  - redis
+---
+
+When you create a Redis™ Database Instance, a single, default database user is a created with [set permissions](/managed-databases/redis/reference-content/default-user-permissions).
+
+You can specify a name for this user when you create a Database Instance via the Scaleway API or the CLI. This allows you to ensure the username is compatible with different software integrations where specific usernames are not supported.
```

Comment:
Actually, you can specify the user_name in the console at the cluster creation. What is not possible is to update it after all in the console. Then you have to use the CLI or the API.

Reply:
```suggestion
You can specify a name for this user when you create a Database Instance. This allows you to ensure the username is compatible with different software integrations where specific usernames are not supported.
```